apps/mobile/src/state/chat/atoms/blockChatActionAtom.ts

```diff
@@ -76,6 +77,11 @@ export default function blockChatActionAtom(
           ...old,
           messages: [successMessage],
         }))
+
+        if (chat.origin.type === 'theirOffer') {
+          set(createSingleOfferReportedFlagAtom(chat.origin.offerId), true)
```

Comment:
Request to flag the offer is not sent. The offer is just marked as flagged, to exclude it from the list.